fw/apm.c

```diff
@@ -1752,18 +1888,44 @@ tfw_cfgop_apm_cleanup_server_failover(TfwCfgSpec *cs)
 	tfw_hm_codes_cnt = 0;
 }
 
+
+static int
+tfw_cfgop_apm_health_stat_srv(TfwCfgSpec *cs, TfwCfgEntry *ce)
+{
+	int i, code;
+	const char *val;
+	TfwApmHMCfg *hm_entry;
+
+	TFW_CFG_CHECK_VAL_N(>, 0, cs, ce);
+	TFW_CFG_CHECK_NO_ATTRS(cs, ce);
+
+	TFW_CFG_ENTRY_FOR_EACH_VAL(ce, i, val) {
+		if (tfw_cfgop_parse_http_status(val, &code))
```

Comment:
Just wanted to bring attention to this line. The function parses HTTP status, and then places the result in some "code" variable. I think that naming of a variable is important in understanding the code (no, I don't mean the `code` variable! :-)) when trying to follow the logic. Just `code` is very ambiguous and can be anything. On the other hand, for instance, `status` is quite specific when linked with `HTTP`. See a discussion in another comment.